docs-ref-conceptual/get-started-tutorial-0-before-you-begin.md

```diff
@@ -0,0 +1,88 @@
+
+> [!div class="checklist"]
+>
+> * 1 - Get to know the Azure CLI in this tutorial step
+> * 2 - [Prepare your environment for the Azure CLI](./get-started-tutorial-1-prepare-environment.md)
+> * 3 - [Learn Azure CLI syntax differences in Bash, PowerShell and Cmd](./get-started-tutorial-2-environment-syntax.md)
+> * 4 - [Populate variables for use in scripts](./get-started-tutorial-3-use-variables.md)
+> * 5 - [Delete Azure resources at scale using a script](./get-started-tutorial-4-delete-resources.md)
+
+## What is the Azure CLI?
+
+The Azure Command-Line Interface (CLI) is a cross-platform command-line tool to connect to Azure and execute administrative commands on Azure resources. It allows the execution of commands through a terminal using interactive command-line prompts or a script.
+
+For interactive use, first launch a shell such as Bash, PowerShell, or Cmd.exe on Windows, Linux or macOS.  Then issue a command at the shell prompt. To automate repetitive tasks, assemble the Azure CLI commands into a shell script using the script syntax of your chosen shell, and then execute the script.
+
+Install the Azure CLI locally on Linux, macOS, or Windows computers. Azure CLI can also be used from a browser through Azure Cloud Shell or run from inside a Docker container.
+
+## What shell environment should I use?
+
+Windows PowerShell, PowerShell, Cmd, and Bash are shell environments. Your shell environment not only determines which tools you can use, but your shell also changes your command-line experience. For example, there are important quoting differences, line continuation characters, and variable syntax between shells.
+
+The Azure CLI reference examples in this tutorial are written for Bash, PowerShell and Cmd to help you evaluate each shell experience.
+
+## How does the Azure CLI compare to Azure PowerShell?
+
+Azure CLI and [Azure PowerShell](/powershell/azure/) are both command-line tools that enable you to create and manage Azure resources, but they are distinctly different.
+
+Here's a brief comparison of the two tools:
+
+| Compare | Azure CLI | Azure PowerShell |
+|-|-|-|
+|What is it?|Cross-platform **command-line interface** | Cross-platform **PowerShell module**
+|Built on | Python | PowerShell
+|Install on| Windows, macOS, Linux | Windows PowerShell, PowerShell
+|Run in|Windows PowerShell, Cmd, Bash, Azure CLoud Shell, Docker container, and other Unix shells.| Windows PowerShell, PowerShell, Azure Cloud Shell, Docker container
+|Terms| Reference groups, reference subgroups and reference commands | Modules, cmdlets and aliases
+|Syntax example | [az storage account create](/cli/azure/storage/account#az-storage-account-create) | [New-AzStorageAccount](/powershell/module/az.storage/new-azstorageaccount) |
+|Support lifecycle| [Azure CLI Support Lifecycle](./azure-cli-support-lifecycle.md) | [Azure PowerShell Support Lifecycle](/powershell/azure/azureps-support-lifecycle)
```

Comment:
This link is broken for me

Reply:
Correct, the link is actually empty.  That article is still a WIP.